src/shared/services/config.service.ts

```diff
@@ -129,7 +129,7 @@ export class ConfigService {
             transports: [
                 new DailyRotateFile({
                     level: 'debug',
-                    filename: `./logs/${this.nodeEnv}/debug-%DATE%.log`,
+                    filename: `${process.env.HOME}/${this.nodeEnv}/debug-%DATE%.log`,
```

Comment:
we need to find a better location for this.

we can't keep `./logs/*` and mount it in docker to be consistent? :thinking: (from developer perspective who uses docker and local env interchangeably)

Reply:
We can play with an env var and default it to env.home if not defined  what do you think about it ?